Model/Config.php

```diff
@@ -24,6 +24,7 @@ class Config
     private const PATH_INTEGRATION_API_URL = 'checkout/bold_checkout_payment_booster_advanced/api_url';
     private const PATH_LOG_IS_ENABLED = 'checkout/bold_checkout_payment_booster_advanced/log_enabled';
     private const PATH_SHARED_SECRET = 'checkout/bold_checkout_payment_booster/shared_secret';
+    private const PATH_PAYMENT_BOOSTER_FLOW_ID = 'checkout/bold_checkout_payment_booster/payment_booster_flow_id';
```

Comment:
[Clarification] I went back and forth on `payment_booster_flow_id` and `default_flow_id` but I think this is clearer, especially if other flows are introduced. Let me know if there's additional thoughts here.

Reply:
This is a great call out. 
I renamed it all to be `bold_booster` just to be consitent. 
Done here: ed9d1e5d7b7eb0513b378bedba33f1c9e9cbbba8 and here: aa068616820a4a7a448693ad638574ac401b24bd

Reply:
Sounds good!